src/redux/configureStore.js

```diff
@@ -0,0 +1,17 @@
+import { combineReducers, legacy_createStore as createStore, applyMiddleware } from 'redux';
+import logger from 'redux-logger';
+import thunk from 'redux-thunk';
+import { composeWithDevTools } from 'redux-devtools-extension';
+import booksReducer, { fetchBooks } from './books/books';
+
+const reducer = combineReducers({
+  booksReducer,
+});
```

Comment:
- [ ] `booksReducer` and `categoriesReducer` should be added to `combineReducers`